src/Form/MenuPositionRuleForm.php

```diff
@@ -262,6 +262,7 @@ protected function getPluginDefinition($menu_name, $parent) {
     $definition['route_name'] = null;
     $definition['route_parameters'] = [];
     $definition['enabled'] = false;
+    $definition['metadata'] = array('entity_id' => $rule->getId());
```

Comment:
Nevermind, we know what menu link plugin it's using so we don't need it to be unique.